backend/compact-connect/stacks/ui_stack/distribution.py

```diff
@@ -75,7 +75,7 @@ def __init__(
                 scope,
                 'CSPFunction',
                 code=Code.from_inline(f.read()),
-                runtime=Runtime.NODEJS_22_X,
+                runtime=Runtime.NODEJS_20_X,
```

Comment:
We upgraded dependencies in other feature branches. This branch is still on a version that doesn't have 22.X added. I opted to bump it back down rather than run the upgrades here as well and produce a bunch of merge conflicts.